package-lock.json

```diff
@@ -7,6 +7,9 @@
     "": {
       "name": "@mattermost/webapp",
       "version": "5.36.0-0",
+      "workspaces": [
```

Comment:
Are the changes in this file intended? It's seems to be conflicting with master

Reply:
nope, not intented at all. Will remove this. Thanks, nice catch.